itests/eth_transactions_test.go

```diff
@@ -131,7 +131,7 @@ func TestLegacyTransaction(t *testing.T) {
 
 	// This is a legacy style transaction obtained from etherscan
 	// Tx details: https://etherscan.io/getRawTx?tx=0x0763262208d89efeeb50c8bb05b50c537903fe9d7bdef3b223fd1f5f69f69b32
-	txBytes, err := hex.DecodeString("f86f830131cf8504a817c800825208942cf1e5a8250ded8835694ebeb90cfa0237fcb9b1882ec4a5251d1100008026a0f5f8d2244d619e211eeb634acd1bea0762b7b4c97bba9f01287c82bfab73f911a015be7982898aa7cc6c6f27ff33e999e4119d6cd51330353474b98067ff56d930")
+	txBytes, err := hex.DecodeString("f901130b8505d21dba008347e7c494c0ee9db1a9e07ca63e4ff0d5fb6f86bf68d47b89890775ec7b96add6c8f0b8a4c4463c80000000000000000000000000000000000000000000000000000000000000003b000000000000000000000000b656b2a9c3b2416437a811e07466ca712f5a5b5a000000000000000000000000f835a0247b0063c04ef22006ebe57c5f11977cc40000000000000000000000000000000000000000000000000000000000000009000000000000000000000000f35e2cc8e6523d683ed44870f5b7cc785051a77d1ba063e1d39bdc0fa652e6d1ab267c9b4d24ead4182296ec51d89950ec1db6aeb842a01db2e243342fc08605e1d353af7f9fd4607e9bc4cef1b37a37d240f001bb7ed2")
```

Comment:
Something is messed up here as looks like we need to do some XOR magic with the V value before submitting this to get the signature verification to work.

Look at what we do in all the other itests above 

```
client.EVM().SignTransaction(&tx, key.PrivateKey)
	// Mangle signature
	tx.V.Int.Xor(tx.V.Int, big.NewInt(1).Int)
```

I am not sure why we need this. Waiting for @vmx to reply on the corresponding Slack thread.

Reply:
The V,R,S values are exactly what they should be (decoded tx using https://rawtxdecode.in/) and yet, the signature verification fails with "failed to calculate sender: invalid signature recovery id".

Something is up. Let me dig.